config/Sage/game_options.txt

```diff
@@ -99,7 +99,7 @@ PROBABILITY TRAITORCHAN 5
 PROBABILITY CHANGELING 5
 PROBABILITY REVOLUTION 4
 PROBABILITY NUCLEAR 4
-PROBABILITY EXTENDED 4
+PROBABILITY EXTENDED 9
```

Comment:
Two things. One, keep it the list ordered. Two. I agree with the others, 9 is way too high. I think 7 would likely be a bit better.